modules/cert-manager-enable-metrics.adoc

```diff
@@ -26,11 +26,43 @@ $ oc label namespace cert-manager openshift.io/cluster-monitoring=true
 
 . Create a service monitor:
 
-.. Create a YAML file that defines the `ServiceMonitor` object:
+.. Create a YAML file that defines the `Role`, `RoleBinding` and `ServiceMonitor` objects:
 +
-.Example `service-monitor.yaml` file
+.Example `monitoring.yaml` file
+
 [source,yaml]
 ----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: cert-manager
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+
```

Comment:
```suggestion
---
```
We should use `---` to divide multiple YAML objects definition. Otherwise will lead to syntax error.